queries.sql

```diff
@@ -144,4 +144,68 @@ FROM owners o
 LEFT JOIN animals a ON o.id = a.owner_id
 GROUP BY o.full_name
 ORDER BY animal_count DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
```

Comment:
- You have done great so far 👍 but there are few requirements you need to meet before moving to the next milestone. 👍 

>Who was the last animal seen by William Tatcher?
List all vets and their specialties, including vets with no specialties.

- [ ] Looking at your `queries.sql` file, there seems to be no code implementation for these requirements, I'll suggest you go through these marked requirements and implement them. :+1:  :+1: